modular/manager/manage_task.go

```diff
@@ -757,15 +758,14 @@ func (m *ManageModular) PickVirtualGroupFamily(ctx context.Context, task task.Ap
 		err error
 		vgf *vgmgr.VirtualGroupFamilyMeta
 	)
-
-	if vgf, err = m.virtualGroupManager.PickVirtualGroupFamily(); err != nil {
+	if vgf, err = m.virtualGroupManager.PickVirtualGroupFamily(vgmgr.NewPickVGFByGVGFilter(m.spBlackList)); err != nil {
 		// create a new gvg, and retry pick.
 		if err = m.createGlobalVirtualGroup(0, nil); err != nil {
 			log.CtxErrorw(ctx, "failed to create global virtual group", "task_info", task.Info(), "error", err)
 			return 0, err
 		}
 		m.virtualGroupManager.ForceRefreshMeta()
-		if vgf, err = m.virtualGroupManager.PickVirtualGroupFamily(); err != nil {
+		if vgf, err = m.virtualGroupManager.PickVirtualGroupFamily(vgmgr.NewPickVGFByGVGFilter(m.spBlackList)); err != nil {
```

Comment:
After we have the `atleast one valid GVG in family` rule, even you create a new famlily, it won't be picked.